<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="image_rect" to="/realsense_wrist/depth/image_rect_raw"/>
    <remap from="camera_info" to="/realsense_wrist/depth/camera_info"/>
    <remap from="points" to="/realsense_wrist/depth/points"/>
  </node>
</launch>